lib/web/apiserver.go

```diff
@@ -1502,7 +1502,9 @@ func (h *Handler) siteSessionGenerate(w http.ResponseWriter, r *http.Request, p
 	if err := httplib.ReadJSON(r, &req); err != nil {
 		return nil, trace.Wrap(err)
 	}
-	req.Session.ID = session.NewID()
+
+	// TODO IN 4.2: change from session.NewLegacyID() to session.NewID().
```

Comment:
Can you change the comment prefix to be `DELETE IN 4.2.0`? That's what we try and use across the board to make it easier to grep through source.

Other than that lgtm.